apps/guide/src/content/03-creating-your-bot/03-adding-commands.mdx

```diff
@@ -68,13 +69,21 @@ The simplest way to acknowledge and respond to an interaction is the _`interacti
 <CH.Code>
 
 ```js
+/** @param {import('discord.js').CommandInteraction} interaction */
 export async function execute(interaction) {
 	await interaction.reply('Pong!');
 }
 ```
 
 </CH.Code>
 
+<Alert title="@type and @param tags" type="info">
+	[`@type`](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#type) and
+	[`@param`](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns) tags are a way
+	to annotate type information, we use them here to improve the in-editor experience. They are not required for your
+	code to run, but they provide autocomplete and types for fields and parameters which is useful.
+</Alert>
```

Comment:
```suggestion
<Alert title="@type and @param tags" type="info">
	[`@type`](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#type) and
	[`@param`](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns) tags allow you
	to annotate your code with type information. The tags are not required for your
	code to run but provide autocomplete and type information for fields and parameters, which can majorly improve your developer experience when working with them.
</Alert>
```

Nitpicking on phrasing and reading flow, as per usual. Might need reformatting.
Variants/other phrasing are fine as well, just to put an idea forward.